.eslintrc.cjs

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+module.exports = {
```

Comment:
suggest, use json format

Schema Store provides JSON Schema

see https://www.schemastore.org/json/
see https://json.schemastore.org/eslintrc.json

Reply:
Hmm, that is nice, but was planning on doing some dynamic things here&hellip;

If we can still extend other configs, it may still turn out to be alright to use JSON for this (am thinking about it).